addons/website_sale_delivery/static/src/js/express_checkout_form.js

```diff
@@ -0,0 +1,18 @@
+odoo.define('website_sale.delivery.payment.express_checkout_form', require => {
+    'use strict';
+
+    const paymentExpressCheckoutForm = require('payment.express_checkout_form');
+
+    paymentExpressCheckoutForm.include({
+        /**
+         * Return whether the shipping information is required or not
+         *
+         * @private
+         * @override method from payment.express_form
+         * @return {Boolean} - Whether the shipping information is required or not
+         */
+         _isShippingInformationRequired: () => {
+            return true;
```

Comment:
I don't know how feasible it is but we don't need shipping information if the order only contains services

Reply:
is it possible to know whether the product is a service or not from js actually or not? If so, I'll add a parameter in the method to take that into account

Reply:
We could deduce it from the page, if there are any delivery methods or not.
It's how it is done currently.. Though if you have a better idea I'm all ears

Reply:
I don't use the same logic to compute the delivery methods over express checkout so I can't assume that we don't need a shipping address from that. The only way to do it would be to know in js if the order is service only or not and to give the args to the override but I think it's a bit overkill for this feature.

Reply:
It would be good to note it for the future.
Created: task 2965823